### ISSUE TITLE
[A11y_Bug]: Keyboard will not open "Learn More" links in Issue List when in Checker view

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1771]
### Project

extension-checker - **_Accessibility Checker_** view
v3.1.64 latest from store

### Browser

Chrome on Windows & MacOS
Firefox on Windows & MacOS
Edge on Windows

### Operating System
Windows 11
MacOS 12.7.1 Monterey 

### Assistive technology
- Keyboard ALONE fails, but NOT with either JAWS or NVDA loaded on Windows
- Fails on both  MacOS and Windows
- Fails on both Chrome and FireFox
- Fails the same on Edge on Windows too
- Keyboard operability is NOT an issue (this is rare) when JAWS or NVDA is loaded in Windows
- Does not fail on Assessment view, only Checker view

### Description

Using the keyboard alone, without JAWS or NVDA loaded, the keyboard does not operate (open) the "Learn More" help screen in the list of issues when in the Checker view.

### Steps to reproduce

1. Open a website, Inspect, open the Accessibility Checker view
2. Scan page
3. Tab until landing on the issues list
4. Arrow down until you are on an issue with a Learn More... link
5. Press Tab to put the focus on "Learn More" link
6. Press Enter and/or Space, neither will open the Help

However, it does operate correctly in the **_Accessibility Assessment_** view but fails in the **_Accessibility Checker_** view

Repeat the same steps with either JAWS or NVDA loaded, the keyboard works fine and opens the Help page. 

Keyboard-only and voice-command users will be affected. Most developers are keyboard users. So it is odd that this hasn't been reported earlier.

**Comment by philljenkins** [comment]
![image](https://github.com/IBMa/equal-access/assets/16193609/f441c706-cdf8-4508-8194-e1ec79dc77e7)

**Comment by philljenkins** [comment]
Completed testing on Chrome, Firefox, and Edge
using both Windows and MacOS platforms

**Comment by drjoho** [comment]
Fixed in #1781

**Comment by philljenkins** [comment]
@drjoho I think this is fixed now (as you demonstrated in the standup),
but check in the Checker view too,
and if no longer reproducible, then you can close this issue I would think.

**Comment by philljenkins** [comment]
Using the latest action for Master,
https://github.com/IBMa/equal-access/actions/runs/7506978855,
the keyboard tab key correctly opens the Help and selects the Dom code in the Checker view.

**Comment by ErickRenteria** [comment]
Fixed, closing issue
